lib/perfil/profileConf.dart

```diff
@@ -0,0 +1,275 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/cupertino.dart';
+
+class ProfileConf extends StatefulWidget {
+  @override
+  MapScreenState createState() => MapScreenState();
+}
+
+class MapScreenState extends State<ProfileConf>
+    with SingleTickerProviderStateMixin {
+  bool _status = true;
+  final FocusNode myFocusNode = FocusNode();
+
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return new Scaffold(
+        body: new Container(
+          color: Colors.white,
+          child: new ListView(
+            children: <Widget>[
+              Column(
+                children: <Widget>[
+                  Column(
+                  children: <Widget>[
+                    new Container(
+                    height: 75.0,
+                    color: Colors.lightGreenAccent, // TODO: Adjust title color
```

Comment:
```suggestion
                    color: Colors.lightGreenAccent, // TODO: Adjust title color
```
Lo de los colores se puede arreglar en el sprint de diseño, pero para ponerlos con los colores de la paleta del prototipo se puede mirar aquí: https://api.flutter.dev/flutter/material/Colors-class.html

Reply:
Sí, mejor lo revisamos en el sprint de diseño